src/net_processing.cpp

```diff
@@ -218,7 +251,10 @@ struct Peer {
     /** Work queue of items requested by this peer **/
     std::deque<CInv> m_getdata_requests GUARDED_BY(m_getdata_requests_mutex);
 
-    explicit Peer(NodeId id) : m_id(id) {}
+    Peer(NodeId id, bool blocks_only)
```

Comment:
Why'd you delete the `explicit`?

Reply:
No good reason. I've put it back.

Reply:
`explicit` prevents implicit conversions for _single-argument_ constructors. After the change above, the constructor has 2 arguments. See https://stackoverflow.com/a/121163

Reply:
`explicit` will also prevent implicit conversion for multiple-argument constructors. Is there a reason why {int,bool} should implicitly be converted into Peer?

Reply:
Was this addressed?

Reply:
fixed now